test/example.robot

```diff
@@ -77,4 +75,11 @@ Test user can use Routine Control By Name service on ECU
 Test user can use Diagnostic Session Control service on ECU
     Log    Diagnostic Session Control service
 
-    Diagnostic Session Control    1
\ No newline at end of file
+    Diagnostic Session Control    1
+
+Test user can use Write Data By Name service on ECU
+    Log    Write Data By Name service
+
+    Log    RealTimeClock_Write
+    ${PARAM_DICT}=    Create Dictionary    Day=26    Month=September    Year=2024    Hour=10    Second=45    Minute=0
+    Write Data By Name    RealTimeClock_Write    ${PARAM_DICT}
```

Comment:
Hi Tri,

Please also add the test case for keyword `Write Data By Identifier` as we discussed.

Thank you,
Ngoan